spec/fabricators/meeting_fabricator.rb

```diff
@@ -0,0 +1,13 @@
+Fabricator(:meeting) do
+    name        { Faker::Lorem.sentence }
+    description { Faker::Lorem.paragraph }
+    date_and_time { Time.zone.now+1.week }
+    # sponsor
+    venue { Fabricate(:sponsor) }
+    invitable true
+    # invites_sent
+    slug "some-slug"
+    spaces 20
+    # created_at
+    # updated_at
+end
```

Comment:
There was no meeting fabricator. I'm not 100% clear on the functionality of some of the attributes so I commented them out for now.